### PR TITLE
Update deployment info

### PR DESCRIPTION
**Pull request by ianrscottexp**
Added Features:
- Added DOCUMENTATION where appropriate
- Added deployment_info variable so code can be used from ECE Proxy

Bug Fixes:
- Update Timeouts
- Update method to get Major and Minor version since version can now be in the deployment_info variable
- Change namespace variable name where appropriate
- Remove unneeded variables for clarity

### CODE REVIEW

**Review by glenntron3030 — APPROVED** [review]
LGTM

File: plugins/module_utils/kibana.py
Comment (mgaruccio) [inline plugins/module_utils/kibana.py:108]: So this isn't important right now, but at some point we may want to change the URL here to an input with a default, custom package repos are technically a thing that's supported

File: plugins/modules/elastic_agentlist_info.py
Comment (mgaruccio) [inline plugins/modules/elastic_agentlist_info.py:64]: We really need to get a black formatting pass added as part of our github actions on commit to any branch, but in the meantime, can you get it installed locally? otherwise we end up with weird formatting changes any time anyone else works on the code and it does things like add trailing commas

File: plugins/modules/elastic_agentpolicy_info.py
Comment (mgaruccio) [inline plugins/modules/elastic_agentpolicy_info.py:69]: we should probably look at more tightly-specifying the `deployment_info` object using `options`.  not all that important for us, but will make the code more usable for others later

File: plugins/modules/elastic_savedobject_info.py
Comment (mgaruccio) [inline plugins/modules/elastic_savedobject_info.py:70]: worthwhile to add an alias to `space_id` here just to avoid any potential isues in the future

ex - `space_id=dict(type='str', default='default', aliases=['space'])`

**Review by mgaruccio — APPROVED** [review]
added a couple of comments and some example changes but really the only thing that I'd like to see updated before merge is getting an alias added for the space_id input, that's not a big enough thing to hold this up though, so approved.

**Pull request merged**
